addons/resource.language.en_gb/resources/strings.po

```diff
@@ -12942,7 +12946,13 @@ msgctxt "#24134"
 msgid "We were unable to load your configured language. Please check your language settings."
 msgstr ""
 
-#empty strings from id 24135 to 24992
+#empty strings from id 24135 to 24991
+
+#: xbmc/addons/CGUIViewStateAddonBrowser.cpp
+#: xbmc/filesystem/AddonsDirectory.cpp
+msgctxt "#24992"
```

Comment:
the manage... for video is generic enough. Though i still dislike (against) using "manage..." at all.